.github/workflows/release-branch.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Release: Create release branch"
```

Comment:
yea maybe we could use a different naming 😅 
But the idea is after creating a new branch with the changes, we wait for every checks to finish manually. So I did this because, if in case there is a failure, then we can stop at this point and prevent a release in github